src/main/java/com/cgdpd/library/service/BookService.java

```diff
@@ -24,4 +34,20 @@ public Book createBook(CreateBookRequestDTO requestDTO) {
         var createdBook = bookRepository.save(bookEntity);
         return bookMapper.mapToBook(createdBook);
     }
+
+    @Transactional(readOnly = true)
```

Comment:
What is the need of this annotation? Please do some research on it and explain why we should need it

Reply:
Still relevant